styles/youtube/catppuccin.user.css

```diff
@@ -1099,6 +1099,20 @@
     .yt-comment-interaction-view-model-wiz__comment-content {
       color: @subtext0;
     }
+
+    /* Horizontal list arrows */
+    .arrow.yt-horizontal-list-renderer {
+      background: @surface0;
+    }
+
+    /* Sign in button */
+    .yt-spec-button-shape-next--overlay.yt-spec-button-shape-next--outline {
+      color: @text;
+      border-color: @surface1;
+      &:hover {
+        background: @surface0;
+      }
+    }
```

Comment:
I would apricate it if you could put this with the rest of the other buttons
https://github.com/catppuccin/userstyles/blob/a117bacf6ab120fd98647bafb62532384646daf4/styles/youtube/catppuccin.user.css#L552